### PR TITLE
Feature/to do

### PR DESCRIPTION
**Pull request by george-shammar**
This pull request seeks to add the following features to the default branch:
1. Project creation functionality
2. Task creation functionality

### CODE REVIEW

File: src/project.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+};
+
+const newProject = () => {
+  newProjectButton.addEventListener('click', () => {
+    projectForm.style.display = 'block';
+  });
+
+  const formSubmit = document.getElementById('submit');
+  formSubmit.addEventListener('click', submit);
+};
```
Comment (oloomoses) [inline src/project.js:106]: - [ ] Kindly separate the DOM manipulation from the logic by putting them in separate modules

File: src/task.js [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    let priority;
+
+
+    taskCreation(descriptionTask, dueDateTask, priority, project);
+    newTaskForm.style.display = 'none';
+
+    showTask(project);
+    saveArray();
+  };
+};
```
Comment (oloomoses) [inline src/task.js:232]: - [ ] Kindly separate the DOM manipulation from the logic by putting them in separate modules

File: src/project.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+function myEvent() {
+  const projectsDiv = document.getElementsByClassName('project-div');
+
+  for (let i = 0; i < projectsDiv.length; i += 1) {
+    projectsDiv[i].addEventListener('click', function display() {
+      displayTask(this.id);
+    });
+  }
+}
```
Comment (oloomoses) [inline src/project.js:64]: - [ ] Kindly separate the DOM manipulation from the logic by putting them in separate modules

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hello team, 
You have done a great job on this app :muscle: 
## Highlights
- I am able to create a project :100: 
- I am able to create and delete to-to :+1: 
- Well done on the linter configuration
- Great job on the readme and pull request :clap: 

## To Improve
- [ ] `webpack-dev-server` should be installed in the devDependencies for `npm start` to execute. Kindly add it. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/devdep_requirement.png))
- [ ] Kindly ensure that DOM-manipulations are kept in a separate module from the logic as per the [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png). Check the comments below for more details :arrow_down: 
- [ ] The submit button for edit todo item does not work. Kindly ensure that a user can edit the details of a to-do item ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/ui_requirement.png))
- [ ] It would be better to have the app deployed and add the link to the live version in the readme.


**Kindly make the above changes before requesting another review**

Feel free to reach me on slack or leave a comment while mentioning me and I will  be happy to respond

_Slack: Oloo Moses_

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi @team :slightly_smiling_face: 

You can now go on with the next project. 

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Pull request merged**